openreview/conference/templates/reviewerWebfield.js

```diff
@@ -141,45 +156,29 @@ var getOfficialReviews = function(noteNumbers) {
     return $.Deferred().resolve([]);
   }
 
-  return Webfield.get('/notes', {
-    invitation: getInvitationId(OFFICIAL_REVIEW_NAME, '.*'), tauthor: true
-  }).then(function(result) {
-    return result.notes || [];
-  });
-};
-
-var getNoteInvitations = function() {
-  return Webfield.get('/invitations', {
-    regex: WILDCARD_INVITATION,
-    invitee: true,
-    duedate: true,
-    replyto: true,
-    type: 'notes',
-    details: 'replytoNote,repliedNotes'
-  }).then(function(result) {
-    return result.invitations || [];
+  var promises = _.map(noteNumbers, function(noteNumber) {
```

Comment:
Won't this make thousands of individual API calls? Is that really more efficient that using a regex?

Reply:
Reviewers usually have very few papers to review right? So that's why I decided to use this approach

Reply:
Ah I missed the fact that this was the reviewer console. My bad.